README.md

```diff
@@ -65,6 +65,12 @@ for variable fonts, as it provides choice over the tradeoff between smoothness (
 with overshoots) and local control. Curious people are encouraged to experiment with the
 `mutator` example in the examples directory.
 
+To run the example:
+
+```rust
+cargo run --example mutator | > image.ppm && open image.ppm
```

Comment:
Totally agree it's good to have command line instructions, but some problems. First, as written, the `|` causes an error. Second, this is mac-only (on Windows "start" has similar functionality, but many users won't have a viewer registered for ppm). I think it's fine if it just says `cargo run --example mutator > image.ppm` and leaves the question of how to open the image to the user.

Also, the lang is wrong, it's shell not rust :)

Reply:
Sure, thanks. I made these updates and pushed a new commit.

I only used `rust` for the code block because that is what the Runebender README is using for shell commands.